src/app/register/pages/register/register.component.html

```diff
@@ -1,79 +1,81 @@
-<main class="container">
-  <div class="row space-around">
-    <div class="col l6 m8 s4" [ngClass]="{ 'no-gutters': platform.columns4 }">
-      <!-- MOBILE PRESENTATION -->
-      <mat-vertical-stepper
-        [linear]="isLinear"
-        #stepper
-        *ngIf="platform.columns8 || platform.columns4"
-        (selectionChange)="selectionChange($event)"
-        class="orcid-stepper-wizard"
-      >
-        <mat-step [stepControl]="FormGroupStepA">
-          <ng-template
-            matStepLabel
-            i18n="@@register.personalData"
-            i18n="@@register.personalData"
-            >Personal data</ng-template
-          >
-          <app-step-a [(formGroup)]="FormGroupStepA"></app-step-a>
-        </mat-step>
-        <mat-step [stepControl]="FormGroupStepB">
-          <ng-template matStepLabel i18n="@@register.securityAndNotifications"
-            >Security and notifications</ng-template
-          >
-          <app-step-b
-            [(formGroup)]="FormGroupStepB"
-            [personalData]="personalData"
-          ></app-step-b>
-        </mat-step>
-        <mat-step [stepControl]="FormGroupStepC" #lastStep>
-          <ng-template matStepLabel i18n="@@register.visibilityAndTerm"
-            >Visibility and terms</ng-template
-          >
-          <app-step-c
-            [(formGroup)]="FormGroupStepC"
-            (submit)="register('vertical')"
-            [loading]="loading"
-          ></app-step-c>
-        </mat-step>
-      </mat-vertical-stepper>
+<main>
```

Comment:
id attribute is missing

Reply:
this is because of the merge conflict... solving